rclpy/src/rclpy/_rclpy.c

```diff
@@ -5093,10 +5094,23 @@ PyMODINIT_FUNC PyInit__rclpy(void)
     return NULL;
   }
 
+  RCLError = PyErr_NewExceptionWithDoc(
+    "_rclpy.RCLError",
+    "Thrown when there is an error in rcl.",
+    PyExc_RuntimeError, NULL);
+    if (NULL == RCLError) {
+      Py_DECREF(m);
+      return NULL;
+    }
```

Comment:
@suab321321 I'm sorry, it was hard to tell indent level when I left my suggestion.

The indentation is one level too deep for these four lines.

Reply:
Fixed 261a2dc